.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+version: 2.1
+jobs:
+  deploy:
+    docker:
+      # Upgrade the Dockerfile and push image when the hub az cluster tiller
+      # version upgrades.
+      - image: quay.io/helmpack/kubectl-helm-az:0.1.0
+        # Don't automatically upgrade the Hub monocular chart version for each
+        # new helm/monocular release. Instead assume we'll need testing, and
+        # set the version manually here.
+        environment:
+          CHART_VERSION: 1.4.4
+    steps:
+      - run:
+          name: Authenticate to cluster
+          shell: /bin/sh -eo pipefail
+          command: |
+            az login --service-principal -u $SERVICE_PRINCIPAL_ID -p $SERVICE_PRINCIPAL_PASS -t $SERVICE_TENANT_ID
+            az aks get-credentials -n $CLUSTER_NAME -g $RESOURCE_GROUP
+      - checkout
+      - run:
+          name: Install compare-url dependencies
+          command: apk add --no-cache curl git
+      - compare-url/reconstruct
+      - run:
+          name: Upgrade/install Monocular Helm chart with any Hub changed values
+          shell: /bin/sh -eo pipefail
+          command: |
+            CIRCLE_COMPARE_URL=$(cat CIRCLE_COMPARE_URL.txt)
+            COMMIT_RANGE=$(echo $CIRCLE_COMPARE_URL | sed 's:^.*/compare/::g')
+            echo "Commit range: $COMMIT_RANGE"
+
+            for config in config/helm-hub-values.yaml config/repo-values.yaml repos.yaml; do
+              if git diff $COMMIT_RANGE --name-status | grep "$config" >/dev/null; then
+                echo "Config file changed: $config"
+                export CONFIG_UPDATED=true
+              fi
+            done
+
+            if [ -n "$CONFIG_UPDATED" ]; then
+              export MONGODB_ROOT_PASSWORD=$(kubectl get secret --namespace helm-hub helm-hub-mongodb -o jsonpath="{.data.mongodb-root-password}" | base64 -d)
+              helm init --client-only
+              helm repo add monocular https://helm.github.io/monocular
```

Comment:
In this container the user runs as root. I'm having trouble running `helm init` and `helm repo add` to add the repository. It fails with errors and isn't setting up the right files in `/root/.helm/`.

Reply:
Ah yes, I didn't add `ca-certificates` - good catch. Rolled new image tag and bumped version.